src/languages/llvm.js

```diff
@@ -13,7 +13,7 @@ export default function(hljs) {
   const IDENT_RE = /([-a-zA-Z$._][\w$.-]*)/;
   const TYPE = {
     className: 'type',
-    begin: /\bi\d+(?=\s|\b)/
+    begin: /\b(?:void|i\d+|half|bfloat|float|double|fp128|x86_fp80|ppc_fp128|x86_amx|x86_mmx|ptr|label|token|metadata|opaque)(?=\s|\b)/
```

Comment:
Is there a reason we couldn't use keyword matching for most of these rather than a long regex?

Reply:
It would be great if types other than `i\d+` can be matched without using a long regex, but I'm not sure how to do it.